docs/superpowers/plans/2026-08-01-scriptmapping-related-instance-access.md

```
@@ -0,0 +1,3573 @@
+    }
+  }
+}
+```
+
+## Internal endpoints
+
+Cross-domain reads go over two internal-only routes, excluded from the public Swagger group:
+
+- `GET  api/v{version}/{domain}/workflows/{workflow}/instances/{instance}/internal/related-data`
+- `POST api/v{version}/{domain}/workflows/{workflow}/internal/related-data/batch`
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)